python/setup.py

```diff
@@ -667,6 +686,27 @@ def get_language_extra_packages():
     return list(packages)
 
 
+def get_tools_extra_packages():
```

Comment:
seems like a near copy-paste of `get_language_extra_packages()`. Could you make a generic function? that implements boths?

Reply:
Thanks for review, updated